src/games/strategy/triplea/ai/proAI/data/ProTerritoryManager.java

```diff
@@ -1044,7 +1044,8 @@ private void findBombardOptions(final PlayerID player, final List<Territory> myU
       // Check strafing and using allied attack if enemy capital/factory
       boolean isEnemyCapitalOrFactory = false;
       final TerritoryAttachment ta = TerritoryAttachment.get(t);
-      if ((ta != null && ta.isCapital()) || ProMatches.territoryHasInfraFactoryAndIsLand(player).match(t)) {
+      if (!t.getOwner().isNull()
```

Comment:
Said in another way, in the context of this if statement, this specific null check does not make sense. It would seem that any other similar `if` statement code would need as similar fix as well, thus my suspicion that this NPE check is in the wrong place.